src/components/Dashboard/TodoList.vue

```diff
@@ -146,7 +146,7 @@
                     :ripple=false
                     @click.stop="togglePanel(index)"
                   >
-                    <v-icon left>mdi-information-outline</v-icon>
+                    <v-icon>mdi-information-outline</v-icon>
```

Comment:
Removed `left`:
![image](https://user-images.githubusercontent.com/51458187/177634424-eb67aa96-244e-403c-96a4-cfbcde3efdd5.png)

Kept `left`:
![image](https://user-images.githubusercontent.com/51458187/177634787-d43a04a2-2e88-42b7-8af6-fa5923ac92b1.png)

[Design link](https://preview.uxpin.com/363a49b33bdb2a314ccac0f18daad8969e93b337#/pages/150002369/simulate/sitemap)

The gap between the icon and "View Details" is larger with `left` than without it, so I removed it.